ChangeLog.md

```diff
@@ -17,6 +17,11 @@ See docs/process.md for how version tagging works.
 
 Current Trunk
 -------------
+- Update libcxx and libcxxabi to LLVM 10 release branch (#11038).
```

Comment:
Was this simply not part in the original PR?

Reply:
Yeah, just wasn't mentioned there. I skimmed the changelog now for notable things, since I was updating it anyhow...